File: src/utils/default_values.ts
export const noItemsList = { itemValues: ['<none>'] };

export const TICKS_BY_PAGE = 16;

export const DEFAULT_MAIN_PAGES = 4;

export const MAX_PAGES = 4;

export const bpmValues = {
  default: 120,
  min: 1,
  max: 300,
};

export const SUBTICKS_BY_TICK = 5;

export const QUARTER_TICK = 4 * SUBTICKS_BY_TICK;

export const GENERAL_MAX_TICKS = TICKS_BY_PAGE * MAX_PAGES;

export const SYSTEM_MAX_TICKS = GENERAL_MAX_TICKS * SUBTICKS_BY_TICK;

File: src-tauri/src/utils/default_values.rs
pub const TICKS_BY_PAGE: u16 = 16;

pub const DEFAULT_MAIN_PAGES: u16 = 4;

pub const MAX_PAGES: u16 = 4;

pub const SUBTICKS_BY_TICK: u16 = 5;

pub const QUARTER_TICK: u32 = 4 * SUBTICKS_BY_TICK as u32;

pub const GENERAL_MAX_TICKS: u32 = (TICKS_BY_PAGE * MAX_PAGES) as u32;

pub const SYSTEM_MAX_TICKS: u32 = GENERAL_MAX_TICKS * SUBTICKS_BY_TICK as u32;
